### ISSUE TITLE
Questions about data sets

### ISSUE THREAD

**Issue by lxy51** [cwmok/DIRAC#17]
Hello, I wonder if you can provide the pre-processed data set, as well as the allocation of training set and test set. I am looking forward to your help very much. My email is liuxyjan@mail.sdu.edu.cn
